tldw_Server_API/app/api/v1/endpoints/media/debug.py

```
@@ -0,0 +1,74 @@
+    try:
+        with db.get_connection() as conn:
+            cursor = conn.cursor()
+
+            cursor.execute("SELECT name FROM sqlite_master WHERE type='table'")
+            tables: List[str] = [row[0] for row in cursor.fetchall()]
+
+            def _table_columns(name: str) -> List[str]:
+                try:
+                    cursor.execute(f"PRAGMA table_info({name})")
+                    return [col[1] for col in cursor.fetchall()]
+                except Exception as exc:  # pragma: no cover - defensive path
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)